airflow/www/static/js/dag_dependencies.js

```diff
@@ -37,6 +37,15 @@ const g = new dagreD3.graphlib.Graph()
   })
   .setDefaultEdgeLabel(() => ({ lineInterpolate: 'basis' }));
 
+// Sometimes the data doesn't load when the scheduler isn't running.
+if (!nodes.length) {
+  $('#error').show();
+  $('#error_msg').text(`
+    There was an issue loading dependencies.
+    Make sure you have dags loaded and that everything is running, then try again.
```

Comment:
I'm still not happy with this case when there are no DAGs -- i.e. a fresh install with out any dags, and with load_examples off -- in that case it shouldn't really show an error; it should just show nothing.

Reply:
I agree. I was trying to keep it simple, but that didn't work out. I'll have it check if there are any dags before showing an error.